plugin/plugin_supported.go

```diff
@@ -1,5 +1,7 @@
-//go:build go1.8 && !windows && amd64 && !static_build && !gccgo
-// +build go1.8,!windows,amd64,!static_build,!gccgo
+//go:build (amd64 || arm64) && !static_build && !gccgo
+// +build amd64 arm64
+// +build !static_build
+// +build !gccgo
```

Comment:
Ah, looks like `ltag` doesn't support multiple `// +build` lines 😞 ; 

```
📚 File headers
  /home/runner/work/containerd/containerd/bin/ltag
  plugin/plugin_supported.go
  Error: Process completed with exit code 1.
```

Reply:
https://github.com/containerd/project-checks/issues/5

Reply:
I opened some PRs earlier today to fix those;

- https://github.com/kunalkushwaha/ltag/pull/14
- https://github.com/kunalkushwaha/ltag/pull/13